packages/api/src/routes/medical/document.ts

```diff
@@ -104,7 +107,7 @@ router.post(
       requestId,
     });
 
-    return res.status(OK).json({ docQueryProgress, requestId });
+    return res.status(OK).json({ download: docQueryProgress.download, requestId });
```

Comment:
this isn't right - if a request is already in progress, this may return more than the downlod.

would suggest doing: `return res.status(OK).json({ ...docQueryProgress, requestId });`

Reply:
also to double check - if the request is still in progress when the user does a new POST - it will return the same requestId?

Reply:
Fixed! Now the request ID persists between calls, but once the query is fully complete, triggering a new one will generate a new ID